python/foglamp/services/south/server.py

```diff
@@ -181,9 +180,7 @@ class Server(FoglampMicroservice):
         Returns:
                registration id and message
         """
-        core_management_port = Parser.get('--port')
-        core_management_address = Parser.get('--address')
-        url = 'http://{}:{}/foglamp/interest'.format(core_management_address, core_management_port)
+        url = 'http://{}:{}/foglamp/interest'.format(self._core_management_host, self._core_management_port)
```

Comment:
We should just be able to do send._core_microservice_management_client.register_interest(category, microservice_id)